src/library/impl/meta_reader/view.h

```diff
@@ -222,7 +222,12 @@ namespace xlang::meta::reader
         static byte_view open_file(std::string_view const& path)
         {
 #if XLANG_PLATFORM_WINDOWS
-            file_handle file{ CreateFileA(c_str(path), GENERIC_READ, FILE_SHARE_READ, nullptr, OPEN_EXISTING, FILE_ATTRIBUTE_NORMAL, nullptr) };
+            auto cstr = c_str(path);
+            int len = MultiByteToWideChar(CP_ACP, 0, cstr, path.length() + 1, 0, 0);
```

Comment:
Prefer to use readable variable names, e.g. length (not len), buffer (not buf), and so on.